src/main/play-doc/developer/BundleConfiguration.md

```diff
@@ -187,7 +222,14 @@ components = {
       "logbroker" = {
         bind-protocol  = "tcp"
         bind-port     = 0
-        services      = ["http://:6379/redis"]
+        service-name  = "redis"
+        acls          = [
+          {
+            tcp = {
```

Comment:
It boils down to HAProxy, and I think HAProxy should be able to do so (since `http` is `tcp` underneath).

Having said that if you choose to do so, some of the HTTP specific syntax (i.e. path_beg, path_regex) won't be available within the TCP mode since the packets won't be parsed.

Coming back to this redis example, I thought it's exposed as `tcp` since the `bind-protocol` is `tcp`?